addons/mass_mailing/models/mailing.py

```diff
@@ -518,12 +518,17 @@ def _action_view_traces_filtered(self, view_filter):
 
     def action_view_clicked(self):
         model_name = self.env['ir.model']._get('link.tracker').display_name
+        helper = (
+            _("No %s clicked your mailing yet!", model_name),
+            _("Link Trackers will measure how many times each link is clicked as well as the proportion of TargetModel who clicked at least once in your mailing.")
+        )
         return {
             'name': model_name,
             'type': 'ir.actions.act_window',
             'view_mode': 'tree',
             'res_model': 'link.tracker',
             'domain': [('mass_mailing_id.id', '=', self.id)],
+            'help': '<p class="o_view_nocontent_smiling_face">' + helper[0] + '</p><p>' + helper[1] + '</p>',
```

Comment:
We could probably use string formatting to make it nicer (aka something like ``<p>%(helper_header)s ...``) .